### PR TITLE
[14.0][IMP] agreement_operating_unit, split agreement_serviceprofile_operating_unit

### PR DESCRIPTION
**Pull request by kittiu**
This PR split `agreement_serviceprofile_operating_unit` form `agreement_operating_unit`

### CODE REVIEW

**Comment by kittiu** [comment]
cc @Vimal-OSI @patrickrwilson @newtratip

File: agreement_serviceprofile_operating_unit/README.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
+==============================
+Agreement with Operating Units
```
Comment (patrickrwilson) [inline agreement_serviceprofile_operating_unit/README.rst:2]: Probably should say "Agreement Service Profiles with Operating Units"

File: agreement_serviceprofile_operating_unit/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2021 Open Source Integrators
+# Copyright (C) 2021 Serpent Consulting Services
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Agreement with Operating Units",
```
Comment (patrickrwilson) [inline agreement_serviceprofile_operating_unit/__manifest__.py:5]: Probably should say "Agreement Service Profiles with Operating Units"

**Review by patrickrwilson — APPROVED** [review]
Works and looks good. Just a minor labeling change requested but not a blocker.

**Pull request closed without merging**